libs/island-ui/core/src/lib/DropdownMenu/DropdownMenu.css.ts

```diff
@@ -9,6 +9,11 @@ export const menu = style({
   },
 })
 
+export const scroll = style({
```

Comment:
I think this could be a bit error prone. Let's say you didn't know how many `items` you were rendering in this menu for whatever reason (maybe you're fetching some data from an api or you are conditionally rendering items where some users have a lot of items and others have a few). If you set the `hasScroll` prop on a menu that has a few `items`, the menu would be way to big, i.e.

<img width="178" alt="Screenshot 2022-09-06 at 08 43 27" src="https://user-images.githubusercontent.com/3789875/188589757-6822f0b7-b315-4722-9ca7-9565f87d8920.png">

Also, this is not very mobile friendly.

I think a better solution would be to add the ability to add class names instead of hasScroll, like we are doing in the `Box` component. That way, the person using the component would have control over how it acted under certain conditions.